gdal/frmts/eeda/eedadataset.cpp

```diff
@@ -187,6 +187,14 @@ GDALEEDALayer::GDALEEDALayer(GDALEEDADataset* poDS,
     m_poFeatureDefn->GetGeomFieldDefn(0)->SetSpatialRef(poSRS);
     poSRS->Release();
 
+    {
+        OGRFieldDefn oFieldDefn("name", OFTString);
+        m_poFeatureDefn->AddFieldDefn(&oFieldDefn);
+    }
+    {
+        OGRFieldDefn oFieldDefn("id", OFTString);
```

Comment:
where is this field populated ?
If it is still needed and populated, then drv_eeda.html should mention it

Reply:
Thanks, I misunderstood how this ended up in the output. I added code to populate them and included descriptions in drv_eeda.html.